### ISSUE TITLE
Normalized mask??

### ISSUE THREAD

**Issue by jlevy44** [albu/albumentations#308]
Hi there. Great package. 

I seem to be having a big issue though, and I'm not sure why this is occurring now. I'm feeding in my image and mask into albumentations transformer.

```
albtorch.transforms.ToTensor(normalize=...
```

There is a totensor and normalization portion of the composed transformer. For some reason, my masks are going from (pysnooper):
```
mask = array([[2, 2, 2, ..., 2, 2, 2],       [2, 2, 2,...0],       [0, 0, 1, ..., 0, 0, 0]], dtype=uint8)
```
to 
```
mask = tensor([[[0.0000, 0.0000, 0.0000,  ..., 0.0078, ... 0.0078, 0.0078,  ..., 0.0000, 0.0000, 0.0000]]])
```
Should I separate the normalizer and only apply it to the image pytorch tensor?

**Comment by albu** [comment]
hi, just don't use `ToTensor` we're in process of deprecating it

**Comment by jlevy44** [comment]
Oh, what should I use then for porting to pytorch? Just direct conversion torch.tensor(...,dtype=torch.float) ???

**Comment by jlevy44** [comment]
For some reason my mask is not skipping the normalization step. Actually, I'm not sure what is happening with it. Ideally, I'd like both the mask and image to undergo the same transformations that are spatially focused and not colors, etc.. But I'm finding that not to be the case and am not sure if it is normalization.

**Comment by ternaus** [comment]
Could you please share an example that reproduces the issue?

**Comment by jlevy44** [comment]
Not at the moment. Everything seems to be working on my end though. Thanks for the insight!
